tests/java/verifiers/src/main/java/io/vectorized/compaction/idempotency/App.java

```diff
@@ -0,0 +1,126 @@
+package io.vectorized.compaction.idempotency;
+import static spark.Spark.*;
+
+import com.google.gson.Gson;
+import spark.*;
+
+public class App {
+  public static class InitBody {
+    public String brokers;
+    public String topic;
+    public int partitions;
+    public int key_set_cardinality;
+  }
+
+  public static class Metrics {
+    public long total_writes = 0;
+    public long total_reads = 0;
+    public long min_writes = 0;
+    public long min_reads = 0;
+    public boolean has_reader = false;
+  }
+
+  public class JsonTransformer implements ResponseTransformer {
+    private Gson gson = new Gson();
+
+    @Override
+    public String render(Object model) {
+      return gson.toJson(model);
+    }
+  }
+
+  Workload workload = null;
+
+  void run() throws Exception {
+    port(8080);
```

Comment:
maybe we can make the port configurable via command line arguments ?

Reply:
I'm not fond of adding things just in case, let's postpone it unless you have a use case for that at hand